api/mapping/models.py

```diff
@@ -191,6 +196,19 @@ class ScanReport(BaseModel):
         blank=True,
     )
 
+    visibility = models.CharField(
+        max_length=10,
+        choices=VisibilityChoices.choices,
+        default=VisibilityChoices.PUBLIC,
+    )
+
+    viewers = models.ManyToManyField(
+        settings.AUTH_USER_MODEL,
+        related_name="scanreport_viewings",
+        related_query_name="scanreport_viewing",
+        blank=True,
```

Comment:
ok so this will require some migrations / manipulation. on the current DB if we not going to allow already uploaded ScanReports to have these as fields (i.e. no `null=True` and `blank=True` ) ? 

If so, with is `viewers` allowed to be blank? That'd be no viewers of a scan report ?

Reply:
On this one there's no `null=True` because it doesn't work with ManyToManyField. This doesn't mean a default is needed though. But this is just the behaviour of this kind of field. Normally you would do something like `null=True` with `blank=True`.

RE: the migrations. They've already been done on model-test-db and they went just fine. We need to do the migrations on the dev db now.

Reply:
ahh interesting to know.

And for migrations, will you go ahead and do them yourself once this is merged? If so, please do.

Reply:
Sure I'll do them :)

Reply:
@calmacx Bit of a pickle. I can't do the migrations on dev. I don't seem to have permission. @spco should be able to help. Let's not merge this branch yet.

Reply:
thanks, im not too sure who can/cant. I also got a bit lost on our workflow here to be honest when it comes to migrations... if it'd be better to do migrations on dev first (and let people know) ; then deploy and migrate to test for the web-tool to be smoke tested